### DISCUSSION TITLE
Cheat

### DISCUSSION THREAD

**Discussion by K420K** [bucanero/apollo-ps4#113] [Q&A] [answered]
Hi I hope you can help me,how can I put offsets for cheats to be injected into the save file?

**Comment by bucanero** [comment, marked as answer]
you need to create a `TITLEID.savepatch` file, and then add the codes in Save Wizard or BSD script format.

For more details check this message: https://github.com/bucanero/apollo-ps4/issues/5#issuecomment-1013883940
